include/deal.II/base/utilities.h

```diff
@@ -1252,7 +1252,10 @@ namespace Utilities
       const std::vector<char>::const_iterator &cend,
       std::vector<T> &                         object)
     {
-      // First get the size of the vector, and resize the output object
+      // The size of the object vector can be found in cbegin of the buffer
+      // The data can be found in [cbegin+1, cend) of the buffer
```

Comment:
Is this comment correct?

Reply:
It is slightly wrong: the data starts at cbegin + sizeof(vector_size).

Reply:
How about now?

This new comment is:
```c++
// The size of the object vector can be found in cbegin of the buffer.
// The data starts at cbegin + sizeof(vector_size).
```